### PR TITLE
One outcome per path for Jordan

### PR DESCRIPTION
**Pull request by chao-xian**
## Trello card

[Create one outcome per path for Jordan marriage abroad](https://trello.com/c/0pmBKZWJ)

## Motivation

Marriage abroad is the most complex Smart Answer to update. It's also one of the most frequent Smart Answers to be updated. We want to simplify the updates so they are quicker, less complex and reduce the need for Developers to make the update. By creating one template per outcome for a country we greatly reduce the amount of logic in the templates.

### Expected changes

As this is a refactor of Jordan, and not a change to the content or flow of the smart-answer, users should not experience any changes. The changes that have occurred to the code are:

Create one outcome per path through the flow for Jordan. This results in 18 new outcomes.
Remove references to Jordan in other outcomes.
Remove Jordan from all country lists.

### Developer note

Jordan did not have a template of its own and was included in a template for several countries, with outputs determined by evaluating in the `MarriageAbroadDataQuery` calculator or from logic in the template.

### CODE REVIEW

File: lib/smart_answer_flows/marriage-abroad/outcomes/countries/jordan/ceremony_country/partner_british/_opposite_sex.govspeak.erb [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+Make an appointment at the local British embassy or consulate to swear an affidavit (written statement of facts) that you’re free to marry. You’ll need to bring the single declaration, your passport, your partner’s passport and pay a fee.
+
+^Documents must be originals if you are getting married at the Sharia Court.^
+
+For non-Islamic marriages, you’ll be asked to provide a certificate of no impediment (CNI) to prove you’re allowed to marry.
+
+To get a CNI, you must post notice of your intended marriage in Jordan. You can do this at the British embassy or in front of a notary public.
+
+$A
```
Comment (pmanrubia) [inline lib/smart_answer_flows/marriage-abroad/outcomes/countries/jordan/ceremony_country/partner_british/_opposite_sex.govspeak.erb]: Could you please try to reuse the partial to render embassies?

**Comment by chao-xian** [comment]
@pmanrubia have re-used the `contact_method` partial as that partial name made more sense in the context of the page, whereas the `overseas_passport` one (while used underneath) felt slightly confusing to me.

File: test/artefacts/marriage-abroad/jordan/ceremony_country/partner_british/opposite_sex.txt [hunk 2]
```diff
@@ -1,5 +1,6 @@
 Marriage in Jordan
 
+
```
Comment (ikennaokpala) [inline test/artefacts/marriage-abroad/jordan/ceremony_country/partner_british/opposite_sex.txt]: This line needs to be deleted

**Comment by chao-xian** [comment]
@ikennaokpala addressed and fixed up

**Comment by ikennaokpala** [comment]
@chao-xian sure will take a look

**Pull request merged**